Website/src/rendering/src/components/Products/ProductDetailsContent.tsx

```diff
@@ -148,14 +150,34 @@ const ProductDetailsContent = ({
     async (e: FormEvent) => {
       e.preventDefault();
       setLoading(true);
-      await dispatch(
+      const response = await dispatch(
         createLineItem({
           ProductID: product.ID,
           Quantity: quantity,
           Specs: specValues,
         })
       );
       setLoading(false);
+
+      const resPayload: { LineItems?: LineItem[] } = response?.payload;
+      const lineItem = resPayload?.LineItems[resPayload.LineItems.length - 1];
+      const addToCartPayload: Partial<AddToCartPayload> = {
+        // TODO change when possible to select language from dropdown
+        language: 'EN',
+        currency: product.xp.Currency,
+        product: {
+          type: product.xp.ProductType.toUpperCase(),
+          item_id: lineItem.Variant?.ID || lineItem.ProductID,
+          name: lineItem.Product.Name,
+          orderedAt: lineItem.DateAdded,
+          quantity: lineItem.Quantity,
```

Comment:
When the product was already in the cart and more of the same product is added to the cart, is the lineItem.quantity the total in the cart or the quantity added in this "add to cart" action? We should only log the additional items to CDP, not the new total quantity.